.github/workflows/ci_linter_run.yml

```diff
@@ -0,0 +1,29 @@
+name: Check Golang code with linter
+
+on:
+  pull_request:
+    branches:
+      - embed-code-go
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22.1'
+
+      - name: Install golangci-linter
+        run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.60.3
+
+      - name: Verify installation
+        run: golangci-lint --version
+
+      - name: Run linters
+        run: |
+          cd ./embed-code-go/
```

Comment:
Try to avoid `cd` if we can achieve something with the `working-directory` attribute.

Reply:
Updated.